lib/backend/k8s/conversion/conversion.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/projectcalico/libcalico-go/lib/names"
 	cnet "github.com/projectcalico/libcalico-go/lib/net"
 	"github.com/projectcalico/libcalico-go/lib/numorstring"
-	extensions "k8s.io/api/extensions/v1beta1"
+	netv1 "k8s.io/api/networking/v1"
```

Comment:
Unfortunately, I think we need to revert back to extensions... We will switch eventually but this breaks support for earlier k8s versions.

KDD only supports v1.7+ anyway, but etcd mode uses this code as well and needs to support v1.6 as well.

Reply:
Got it. Good thing I didn't squash the markups commit yet. Will revert and resend.